specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -4,9 +4,11 @@ import "@azure-tools/typespec-azure-core";
 
 using TypeSpec.Rest;
 using TypeSpec.Http;
+using TypeSpec.Versioning;
 
 namespace AzureLoadTesting;
 
+@doc("Metrics for pass/fail criteria.")
```

Comment:
nit: should we use `-` in place of `/` ?
`pass-fail`

Reply:
Will confirm with Nikita once.

Reply:
We have used pass/fail in GA version also. Going with pass/fail in this one too